lib/types/BaseAssetContent.ts

```diff
@@ -16,4 +16,6 @@ export interface BaseAssetContent extends KDocumentContent {
   measures?: MeasureContent[],
 
   metadata?: JSONObject,
+
+  deviceIds?: string[]
```

Comment:
This should always be set as an empty array at least

Reply:
Done